harmonisation/intro.md

```diff
@@ -0,0 +1,21 @@
+# Introduction to harmonisation
+
+For DiSSCo to provide be fully machine actionable units (FDOs) there needs to be a standardized datamodel.
+This datamodel provides a way to know which information to expect and where to expect it.
+The data standard we will call the open Digital Specimen (openDS) and it build upon existing standards.
```

Comment:
This sentence also is weird to read, do you mean: 'The data standard, which we will call the open Digital Sepcimen (openDS), builds upon the existing standards.'

Reply:
Changed